tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_creator.py

```diff
@@ -27,6 +27,10 @@
 from tensorflow.python.training.saver import BaseSaverBuilder
 from tensorflow_recommenders_addons import dynamic_embedding as de
 
+KHkvHashTableInitCapacity = 1024 * 1024
+KHkvHashTableMaxCapacity = 1024 * 1024
+KHkvHashTableMaxHbmForValues = 1024 * 1024 * 1024
```

Comment:
It would be better to comment the `by bytes` at the end of the line or the var name.

Reply:
DONE